internal/utils.go

```diff
@@ -183,6 +191,26 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 	return policyTemplates, nil
 }
 
+// checkPolicyTypeManifest determine if the manifest is policy-type manifest.
+func checkPolicyTypeManifest(manifest map[string]interface{}) bool {
+	apiVersion, foundAPIVersion, isStr := unstructured.NestedString(manifest, "apiVersion")
+	if !foundAPIVersion || isStr != nil {
+		return false
+	} else if strings.HasPrefix(apiVersion, "policy.open-cluster-management.io") {
```

Comment:
This has the potential to nest a `Policy`, `PolicySet`, etc. kind inside of a policy, which I'm not sure we'd want. Should we check against a list of supported kinds?

@mprahl @ycao56 What do you think?

Reply:
I would also like the `Kind` to be checked to see if the string ends with `Policy` and also doesn't equal `Policy`.

Reply:
That's a good idea--then we wouldn't have to maintain a list of policy kinds.